QEfficient/generation/text_generation_inference.py

```diff
@@ -460,6 +470,10 @@ def prepare_decode_inputs(self):
         if self.batch_index is not None:
             decode_inputs["batch_index"] = self.batch_index
 
+        if self.prompt_to_lora_id_mapping_decode and self.full_batch_size is not None:
```

Comment:
is it possible to run lorax in non-CB mode?

Reply:
It only support CB for now as the export and compile is implement for CB only.

Reply:
Will have to test it and update the code.